notebooks/s3_communication.py

```diff
@@ -0,0 +1,146 @@
+"""S3 communication tools."""
```

Comment:
Off Topic: Is this something we should put in an OSG-utils (or some other name) package on a package index to make more easily usable?  Seems fairly generic and applicable to multiple projects. WDYT?

Reply:
Yes, we think that's a good idea. @chauhankaranraj let's set time to do this finally?